src/lib/airtable/tables/__schemaGen.js

```diff
@@ -0,0 +1,30 @@
+// You can enter this in the console on:
```

Comment:
im a little confused about that the inputs and outputs/sideeffects of this script is, can you elaborate?

Reply:
For sure. I'll add some more docs and work it around a bit. Very goofy hack

Reply:
done